lm_eval/jasquad/jasquad.py

```diff
@@ -105,17 +105,5 @@ def _compute(self, predictions, references):
                 ]
             }
         ]
-        
```

Comment:
I don't understand it deeply but do we need to remove because of this change...? I felt it's cool to make it faster but I would be sad if we need to remove it lol

Reply:
Yes I just didn't want to make the review more complicated for you but I did like this so I will make some small changes to make it back ⚙️

Reply:
Thank you!